lab-guides/01-AzureLocalCloudDeployment/readme.md

```diff
@@ -435,7 +435,6 @@ Since we already have credentials and TrustedHosts configured in Powershell from
 #install modules (Not needed with latest media
 <#
 Invoke-Command -ComputerName $Servers -Scriptblock {
-    Install-PackageProvider -Name NuGet -MinimumVersion 2.8.5.201 -Force
```

Comment:
according to docs this seems to be part of the ISO.  I have tested this and it seems in fact not required anymore in 2507 and 2508